docs/reference/commandline/version.md

```diff
@@ -25,9 +25,9 @@ number of the Docker CLI you are using.
 ### Default output
 
 The default output renders all version information divided into two sections;
-the "Client" section contains information about the Docker CLI and client
-components, and the "Server" section contains information about the Docker
-Engine and components used by the Engine, such as the "Containerd" and "Runc"
+the `Client` section contains information about the Docker CLI and client
+components, and the `Server` section contains information about the Docker
+Engine and components used by the Docker Engine, such as the `containerd` and `runc`
```

Comment:
here we're referring to c8d and runc as runtime programs, not binaries, so I don't think they should be stylized like code

```suggestion
Engine and components used by the Docker Engine, such as the containerd and runc
```